code/game/gamemodes/changeling/changeling.dm

```diff
@@ -359,7 +359,7 @@ GLOBAL_LIST_INIT(slot2type, list("head" = /obj/item/clothing/head/changeling, "w
 			return
 	if(!target)
 		return
-	if(target.disabilities & HUSK)
+	if((target.disabilities & NOCLONE) || (target.disabilities & HUSK))

```

Comment:
Not a fan of this. An IPC's DNA isn't "ruined beyond usability", they just don't have it to start with.

Reply:
IPCs are `NOSCAN`, not `NOCLONE`, per our discussion in discord.